Unused variable `tmp`

Seeing the following warning:

    FrequencyTimer2.cpp:351:10: warning: variable 'tmp' set but not used [-Wunused-but-set-variable]
    uint8_t tmp = CMT_MSC;

Running Teensyduino 1.27, Arduino 1.6.7 on OS X 10.11.4.

--- a/FrequencyTimer2.cpp
+++ b/FrequencyTimer2.cpp
@@ -348,7 +348,7 @@ void cmt_isr(void)
 {
 	static uint8_t inHandler = 0;
 
-	uint8_t tmp = CMT_MSC;
+	uint8_t __attribute__((unused)) tmp = CMT_MSC;
 	tmp = CMT_CMD2;
 	if ( !inHandler && FrequencyTimer2::onOverflow) {
 		inHandler = 1;

